lib/minfraud/minfraud_client.rb

```diff
@@ -69,13 +67,14 @@ def initialize(
 
     # @param url [String] URL endpoint, such as '/app/ccv2r'
     # @param transaction [Minfraud::Transaction] transaction to be sent to MaxMind
+    # @yield [Minfraud::Response] a block to pass to the response to allow for modification
```

Comment:
Is this modification on the response itself? Isn't it immutable?

Reply:
This happens during the initialization of the Response (before it becomes immutable) - updated the comment to make this more clear!